### DISCUSSION TITLE
如何修改request get的请求头

### DISCUSSION THREAD

**Discussion by diandianti** [tw93/Pake#536] [Q&A]
我尝试打包网页版的微信，但是登录失败，怀疑是因为请求头没有修改

因为浏览器中最开始登录失败，然后我在 浏览器中开启这个插件 https://github.com/adamyi/wechrome 就可以登录成功

**Comment by Tlntin** [comment]
https://github.com/tw93/Pake/blob/master/src-tauri/pake.json#L13

你说的应该是这个。

**Reply by diandianti** [reply]
这个地方只能配置header里面的UA段，但是除了这部分还需要其他的配置。
